packages/block-editor/src/components/colors/use-colors.js

```diff
@@ -194,6 +194,7 @@ export default function __experimentalUseColors(
 						[ color ? name : camelCase( `custom ${ name }` ) ]: color ?
 							color.slug :
 							newColor,
+						[ camelCase( `value ${ name }` ) ]: newColor,
```

Comment:
This line will set the `value${ name }` attribute. It allows getting the color value in the `rgb()` notation, very useful when we need to apply them using inline styles beyond how the color has been defined (through the colors palette, custom color).
It's already handled by the hook, populating the `style` attribute of its children component, for instance...

```jsx
<TextColor>
    <MyComponent />
</TextColor>
```

The style attribute of the `<MyComponent />` will get populated by the parent `<TextColor />` one.

The problem is, on this implementation, in particular, we need to know the `rgb()` value for the inner blocks (NavigationItem). We need the ability to get these values, store them in the Navigation attributes, to finally collect them from the `<NavigationItem />` using the `getBlockAttributes()` store selector.

Pinging to @epiqueras to get a qualified opinion about this change.

Reply:
You can just use `TextColor.color`.

Reply:
> You can just use `TextColor.color`.

Yup, I already know that it's possible to pick up the property from the component.
Anyway, we need to store these values into the attributes object since it's needed to pull them from the inner blocks.
Thanks for your advice!